content/reference/api/http.md

```diff
@@ -2211,6 +2242,7 @@ Get and serialize the DAG node named by <key>.
 #### Arguments
 
   - `arg` [string]: Key of the object to retrieve, in base58-encoded multihash format. Required: **yes**.
+  - `data-encoding` [string]: Encoding type of the data field, either "text" or "base64". Default: "text". Required: no.
```

Comment:
Do we know why default in go-ipfs (text) is different than one in js-ipfs ([base64](https://github.com/ipfs/js-ipfs/blob/323b0dc55624b0732d006a3fc6bc7a7dc3d263d0/src/cli/commands/object/get.js#L13))?
Which one is safer?

cc @achingbrain @alanshaw

Reply:
base64 is safer as go’s text encoding can generate bytes sequences that are not valid in JSON strings.

text is the default for go as that’s what it’s always been. Should probably change.